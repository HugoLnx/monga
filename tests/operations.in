void main() {
	int a,b,c;
	a = 10;
	b = 20;
	c = a + b;
	printf("c: %d\n", c);
	printf("a+b: %d\n", a + b);
	printf("a+b+c: %d\n", a + b + c);
	printf("b-a: %d\n", b - a);
	printf("b/a: %d\n", b / a);
	printf("a*b: %d\n", a*b);
	printf("a+b*c/a: %d\n", a+b*c/a);
}
