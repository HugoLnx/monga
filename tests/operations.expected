c: 30
a+b: 30
a+b+c: 60
b-a: 10
b/a: 2
a*b: 200
a+b*c/a: 70
